config.py

```diff
@@ -71,6 +71,11 @@ class Config(object):
     DM_APP_NAME = 'buyer-frontend'
     DM_DOWNSTREAM_REQUEST_ID_HEADER = 'X-Amz-Cf-Id'
 
+    DM_FRAMEWORK_CONTENT_MAP = {
```

Comment:
An alternative to this (not necessarily a better alternative) might be to try to load the relevant manifest and then default to G6 if it can't be loaded?

Or - and this might be really mad - create G4 and G5 folders in frameworks repo and copy the required G6 manifest there.